geopandas/sindex.py

```diff
@@ -249,11 +318,61 @@ def intersection(self, coordinates):
             coordinates : sequence or array
                 Sequence of the form (min_x, min_y, max_x, max_y)
                 to query a rectangle or (x, y) to query a point.
+
+            Examples
+            --------
+            >>> from shapely.geometry import Point, box
+            >>> s = geopandas.GeoSeries(geopandas.points_from_xy(range(10), range(10)))
```

Comment:
Maybe limit to 5? (since only up to the 4th is returned in the example) That limits the length of showing `s` a bit